core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ReadWriteLDAPUserStoreConstants.java

```diff
@@ -68,47 +70,58 @@
-
-        setProperty(UserStoreConfigConstants.usernameJavaRegEx, "Username RegEx (Java)", "[a-zA-Z0-9._-|//]{3,30}$", UserStoreConfigConstants.usernameJavaRegExDescription);
-        setProperty(UserStoreConfigConstants.usernameJavaScriptRegEx, "Username RegEx (Javascript)", "^[\\S]{3,30}$", UserStoreConfigConstants.usernameJavaRegExDescription);
+        setProperty(UserStoreConfigConstants.membershipAttribute, "Membership Attribute", "member",
+                UserStoreConfigConstants.membershipAttributeDescription);
+        setProperty(UserStoreConfigConstants.memberOfAttribute, "Member Of Attribute", "",
+                UserStoreConfigConstants.memberOfAttribute);
+        setProperty("BackLinksEnabled", "Enable Back Links", "false",
+                "Whether to allow attributes to be result from references to the object from other objects");
+
+        setProperty(UserStoreConfigConstants.usernameJavaRegEx, "Username RegEx (Java)", "[a-zA-Z0-9._-|//]{3,30}$",
+                UserStoreConfigConstants.usernameJavaRegExDescription);
+        setProperty(UserStoreConfigConstants.usernameJavaScriptRegEx, "Username RegEx (Javascript)", "^[\\S]{3,30}$",
+                UserStoreConfigConstants.usernameJavaRegExDescription);
         setProperty(usernameJavaRegExViolationErrorMsg, "Username RegEx Violation Error Message",
                 "Username pattern policy violated.", usernameJavaRegExViolationErrorMsgDescription);
         setProperty(passwordJavaRegEx, "Password RegEx (Java)", "^[\\S]{5,30}$", passwordJavaRegExDescription);
         setProperty(UserStoreConfigConstants.passwordJavaScriptRegEx, "Password RegEx (Javascript)", "^[\\S]{5,30}$",
                 UserStoreConfigConstants.passwordJavaScriptRegExDescription);
         setProperty(passwordJavaRegExViolationErrorMsg, "Password RegEx Violation Error Message",
                 "Password pattern policy violated.", passwordJavaRegExViolationErrorMsgDescription);
-        setProperty(UserStoreConfigConstants.roleNameJavaRegEx, "Role Name RegEx (Java)", "[a-zA-Z0-9._-|//]{3,30}$", UserStoreConfigConstants.roleNameJavaRegExDescription);
-        setProperty(UserStoreConfigConstants.roleNameJavaScriptRegEx, "Role Name RegEx (Javascript)", "^[\\S]{3,30}$", UserStoreConfigConstants.roleNameJavaScriptRegExDescription);
+        setProperty(UserStoreConfigConstants.roleNameJavaRegEx, "Role Name RegEx (Java)", "[a-zA-Z0-9._-|//]{3,30}$",
+                UserStoreConfigConstants.roleNameJavaRegExDescription);
+        setProperty(UserStoreConfigConstants.roleNameJavaScriptRegEx, "Role Name RegEx (Javascript)", "^[\\S]{3,30}$",
+                UserStoreConfigConstants.roleNameJavaScriptRegExDescription);
         setProperty("UniqueID", "", "", "");
 
     }
 
     private static void setMandatoryProperty(String name, String displayName, String value,
                                              String description, boolean encrypt) {
+
```

Comment:
IMO the blank line here should be removed since only the relevant change should be reflected in the PR. WDYT?

Reply:
Okay. I will remove that